MaterialLib/PorousMedium/Permeability/createPermeabilityModel.h

```diff
@@ -0,0 +1,33 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2016, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ * \file   createPermeabilityModel.h
+ *
+ * Created on August 17, 2016, 2:36 PM
+ */
+
+#ifndef OGS_CREATEPERMEABILITYMODEL_H
+#define OGS_CREATEPERMEABILITYMODEL_H
+
+#include <Eigen/Dense>
+
+#include "BaseLib/ConfigTree.h"
+
+namespace MaterialLib
+{
+namespace PorousMedium
+{
+using CoefMatrix = Eigen::MatrixXd;
+
+/// Create a porosity model
+/// \param config  ConfigTree object has a tag of <permeability>
+CoefMatrix createPermeabilityModel(BaseLib::ConfigTree const& config);
```

Comment:
I'd prefer to see the explicit type `Eigen::MatrixXd` instead of a hidden type alias, but this is personal preference.